src/go/rdctl/cmd/snapshot.go

```diff
@@ -24,7 +24,7 @@ var snapshotErrors []error
 
 const backendLockName = "backend.lock"
 
-type cobraFunc func(cmd *cobra.Command, args []string) error
+type cobraFunc func() error
```

Comment:
Now that the callback is just `func () error` I don't think there's a need for a name, which would show up only
in the defn of `wrapSnapshotOperation`

Not naming it is a little longer, but because it's used only once I don't see that we need to use a named type.